File: apps/problems/Sort-Array/boilerplate-full/function.cpp

            #include <stdio.h>
            #include <stdlib.h>
            #include <stdbool.h>
            #include <string.h>
            #include <limits.h>
            
            ##USER_CODE_HERE##
            
            int main() {
                FILE *file = fopen("/dev/problems/Sort-Array/tests/inputs/##INPUT_FILE_INDEX##.txt", "r");
                vector<string> lines;
                string line;
                while (fgets(line, sizeof(line), file)) lines.push_back(line);
                fclose(file);
                int size_values;
  fscanf(file, "%d", &size_values);
  int* values[size_values];
  for (int i = 0; i < size_values; i++) fscanf(file, "%d", &values[i]);
                list<int> result = sortArray(values);
                fprintf(stdout, "%d\n", result);
                return 0;
                }
            

File: apps/problems/Sort-Array/boilerplate-full/function.java

            import java.io.*;
            import java.util.*;

            public class Main {
                
                ##USER_CODE_HERE##

                public static void main(String[] args) {
                    String filePath = "/dev/problems/Sort-Array/tests/inputs/##INPUT_FILE_INDEX##.txt"; 
                    List<String> lines = readLinesFromFile(filePath);
                    int size_values = Integer.parseInt(lines.get(0).trim());

        List<Integer> values = new ArrayList<>(size_values);

        String[] inputStream = lines.get(1).trim().split("\s+");

        for (String inputChar : inputStream)  {

          values.add(Integer.parseInt(inputChar));

        }

                    List<Integer> result = sortArray(values);
                    System.out.println(result);
                }
                public static List<String> readLinesFromFile(String filePath) {
                    List<String> lines = new ArrayList<>();
                    try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
                        String line;
                        while ((line = br.readLine()) != null) {
                            lines.add(line);
                        }
                    } catch (IOException e) {
                        e.printStackTrace();
                    }
                    return lines;
                }
            }